packages/terra-tabs/src/clinical-lowlight-theme/Tabs.module.scss

```diff
@@ -0,0 +1,117 @@
+    --terra-tabs-structural-active-hover-background-size: 100% 2px;
+    --terra-tabs-structural-active-z-index: 0;
+    --terra-tabs-structural-active-before-transition-duration: 0;
+    --terra-tabs-structural-active-before-transition-timing-function: ease;
+
+    --terra-tabs-structural-before-content-height: 0;
+    --terra-tabs-structural-between-tabs-margin-left: 0;
+    --terra-tabs-structural-extended-tabs-border-radius: 0;
+
+    --terra-tabs-structural-focus-background-color: #222a2e;
+    --terra-tabs-structural-focus-color: initial;
+
+    --terra-tabs-structural-hover-background-color: #222a2e;
+    --terra-tabs-structural-hover-background-image: linear-gradient(to bottom, #111212, #111212);
+    --terra-tabs-structural-hover-color: #b2b5b6;
+    --terra-tabs-structural-hover-background-size: 100% 2px;
+    --terra-tabs-collapsed-tab-menu-hover-background-size: 0 0;
+
+    --terra-tabs-active-focus-ltr-box-shadow: 0 0 1px 3px #4a90e2, 0 0 7px 4px #4a90e2;
+    --terra-tabs-active-focus-rtl-box-shadow: 0 0 1px 3px #4a90e2, 0 0 7px 4px #4a90e2;
+
+    --terra-tabs-keyboard-active-focus-ltr-box-shadow: 0 0 1px 3px #4a90e2, 0 0 7px 4px #4a90e2;
+    --terra-tabs-keyboard-active-focus-rtl-box-shadow: 0 0 1px 3px #4a90e2, 0 0 7px 4px #4a90e2;
+
+    --terra-tabs-structural-active-background-image: linear-gradient(to bottom, #007ee0, #007ee0);
+    --terra-tabs-structural-active-focus-background-image: linear-gradient(to bottom, #007ee0, #007ee0);
+    --terra-tabs-structural-active-background-size: 100% 2px;
+    --terra-tabs-structural-collapsed-before-content-background: #141719;
+    --terra-tabs-structural-collapsed-before-content-height: 0;
+    --terra-tabs-structural-collapsed-border-bottom-color: #181b1d;
+    --terra-tabs-structural-collapsed-menu-bar-focus-border-bottom-color: none;
+    --terra-tabs-collapsed-tab-menu-active-background-size: 0 0;
+    --terra-tabs-structural-content-background-image: linear-gradient(to bottom, #222a2e, #222a2e);
+    --terra-tabs-structural-collapsed-border-bottom-width: 1px;
+    --terra-tabs-structural-collapsed-border-radius: 0;
+    --terra-tabs-structural-collapsed-menu-bar-focus-border-bottom-width: 0;
+    --terra-tabs-structural-collapsed-position: static;
+    --terra-tabs-structural-disabled-hover-background-color: #222a2e;
+    --terra-tabs-structural-disabled-hover-color: #404344;
+    --terra-tabs-structural-disabled-hover-opacity: 1;
+    --terra-tabs-structural-focus-background-image: linear-gradient(to bottom, #141719, #141719);
+    --terra-tabs-structural-last-tab-border-width: 0;
+    --terra-tabs-structural-tab-bar-border-bottom-color: #181b1d;
+    --terra-tabs-structural-tab-bar-border-bottom-width: 1px;
+    --terra-tabs-structural-transition-duration: 0;
+    --terra-tabs-structural-transition-timing-function: ease;
+
+    /* NOTE: This is being commented out until discussions have been resolved around if modular tabs should be removed.
```

Comment:
We should probably just remove these commented styles for modular tabs. It's unlikely to even be correct at this point.